moldesign/interfaces/pdbfixer_interface.py

```diff
@@ -97,21 +97,50 @@ def mutate_residues(mol, residue_map):
     # into an MDT structure
     assert temp_mutant.num_residues == mol.num_residues  # shouldn't change number of residues
     residues_to_copy = []
+    old_residue_map = {}
     for oldres, mutant_res in zip(mol.residues, temp_mutant.residues):
         if oldres in residue_map:
             residues_to_copy.append(mutant_res)
             mutant_res.mol = None
             mutant_res.chain = oldres.chain
+            old_residue_map[oldres] = mutant_res
         else:
             residues_to_copy.append(oldres)
 
+    # Bonds between original and mutated backbone atoms will be removed when 
+    # creating the new mutant molecule because the original and mutated atoms 
+    # reference different molecules.
+    #
+    # Make a list of bonds referencing atoms in the original molecule that
+    # is used later to recreate bonds between original and mutated backbone 
+    # atoms.
+    orig_bonds = []
+    for res in residues_to_copy:
+        if res.type not in mdt.data.BACKBONES:
```

Comment:
Would be good to refactor this not to explicitly depend on backbones if possible (although, in practice, it will presumably always be the case).

Probably you will want to use `res.external_bonds` - an iterator over bonds between this residue and another one (for proteins, these are generally backbone bonds + any CYS bridges)